### ISSUE TITLE
bold class

### ISSUE THREAD

**Issue by rimmesbe** [gcraill/prj-rev-bwfs-dasmoto#4]
You generally will not see this done because so many elements come with bold default. If you really wanted text emboldened then you should most likely be using one of the elements that comes with bold default as it is probably important.
`<strong> <h1> <h2> <b> etc.`

https://github.com/gcraill/prj-rev-bwfs-dasmoto/blob/master/S02E06%20Dasmotos/resources/css/index.css#L12-L14

**Comment by gcraill** [comment]
Good point thanks. Is <b> deprecated in favour of <strong> ? or are they both valid?
